src/App.js

```diff
@@ -1,16 +1,68 @@
-import { Route, Routes } from "react-router-dom";
+/** @format */
+
+import { Route, Routes, useLocation } from "react-router-dom";
 import "./App.css";
-import { Landing, Login, Signup } from "./screens";
+import { PrivateRoute } from "./components/PrivateRoute/PrivateRoute";
+import {
+  Landing,
+  Archive,
+  Login,
+  Sidebar,
+  Signup,
+  Home,
+  Lable,
+  Trash,
+} from "./screens";
 
 function App() {
+  const location = useLocation();
+
+  const locationFlag =
+    location.pathname === "/" ||
+    location.pathname === "/login" ||
+    location.pathname === "/signup";
   return (
-    <>
+    <div className={locationFlag ? "" : "main-wrapper"}>
+      {!locationFlag && <Sidebar />}
       <Routes>
         <Route path='/' element={<Landing />} />
+
         <Route path='/login' element={<Login />} />
         <Route path='/signup' element={<Signup />} />
+        <Route
+          path='/home'
+          element={
+            <PrivateRoute>
+              <Home />
+            </PrivateRoute>
```

Comment:
you don't need to repeat private route components for multiple, you can use outlet from router to do achieve the same thing